### ISSUE TITLE
In confluence module missing /wiki in api_root 

### ISSUE THREAD

**Issue by visko-tm** [atlassian-api/atlassian-python-api#189]
I am using atlassian-python-api==1.12.6

This call
```python
pprint(confluence.get_space('MYSPACE'))
```
returns
`Received: 404
 Not Found
{'message': 'null for uri: '
            'https://mycompany.atlassian.net/rest/api/space/MYSPACE?expand=description.plain,homepage',
 'status-code': 404}`

And indeed, the [API docs](https://developer.atlassian.com/cloud/confluence/rest/#api-space-get) says that the api root is **wiki**/rest/api and not rest/api. 
Overriding the api_root did not change anything:
```python
Confluence(username=user,
           password=api_token,
           api_root='wiki/rest/api',
           url='https://mycompany.atlassian.net'
```
But the return value of get_space() is the same.

Could you please check / suggest?

**Comment by gonchik** [comment]
Hi! 

Looks like we need implement api_root correctly.  
I want to play around path api root. 
As workaround you can set into url path as well :)

**Comment by visko-tm** [comment]
Yes, you are right, this way it works:
```python
Confluence(username=user,
           password=api_token,
           api_root='rest/api',
           url='https://mycompany.atlassian.net/wiki'
```
but I think it's a pretty common situation. Would it be possible to default with 'wiki/rest/api' and/or update the documentation with the proper 'url' (extended with /wiki)?

**Comment by gonchik** [comment]
I hope that small hacky fix f16c1dd
will cover your requests
